keps/sig-api-machinery/4020-unknown-version-interoperability-proxy/README.md

```diff
@@ -305,7 +284,7 @@ For the mTLS between source and destination apiservers, we will do the following
 2. The server (destination apiserver) will check the client (source apiserver) certs to determine that the proxy request is from an authenticated client. We will use requestheader authentication (and NOT client cert authentication) for this. The client (source apiserver) will provide the [proxy-client certfiles](https://github.com/kubernetes/kubernetes/blob/release-1.27/cmd/kube-apiserver/app/options/options.go#L222-L233) to the server (destination apiserver) which will verify the presented certs using the CA bundle provided in the [--requestheader-client-ca-file](https://github.com/kubernetes/kubernetes/blob/release-1.27/staging/src/k8s.io/apiserver/pkg/server/options/authentication.go#L125-L128) passed to the apiserver upon bootstrap
 
 ### Discovery Merging
-TODO: detailed description of discovery merging. (not scheduled until beta.)
+Maintaining the Remote Discovery cache, which provides a merged view of all resources available in the cluster at any given time (even during mixed version state), naturally solves the problem of serving a comprehensive discovery document.
```

Comment:
We probably need to elaborate more on the discovery merging mechanism. Is the merged discovery only generated and used by MVP to decide on proxying or would the discovery handling itself need to change to yield a merged document?